src/mainField/MainField.js

```diff
@@ -23,7 +23,8 @@ export const MainField = ({ state, indicators, petName }) => {
     stateIndicator === "dead" ? undefined : stateIndicator
   );
 
-  const illness = getRandomEl(arrIllnesses);
+  // const illness = getRandomEl(arrIllnesses);
```

Comment:
and let's remove all the unnecessary comments. Please also re-check the terminal for any warnings regarding unused variables